content/en/blog/extend-falco-inputs-with-a-plugin-register.md

```diff
@@ -0,0 +1,109 @@
+| 999 | test                                                                                      | `test`           | This ID is reserved for source plugin development. Any plugin author can use this ID, but authors can expect events from other developers with this ID. After development is complete, the author should request an actual ID | Authors: N/A <br/> License: N/A                                                     |
+
+# Details of your plugin
+
+In this section, we'll describe the key elements for describing your plugin.
+
+## License
+
+The license is not mandatory and you're free to choose the one you want for your plugin. Keep in mind it's often a good practice to have a license.
+
+## ID
+
+Every source plugin requires its own, unique plugin event `ID` to interoperate with `Falco` and the other plugins. This `ID` is used in the following ways:
+
+* The `ID` is saved in in-memory event objects and is used to identify the associated plugin that injected the event.
+* The `ID` is saved in capture files and is used to recreate in-memory event objects when reading capture files.
+
+The `ID` must be unique to ensure that events written by a given plugin will be properly associated with that plugin (and its event sources, see below).
+
+## Name
+
+Each plugin in the registry must have its own `name` and can be different from `event source`, that can be shared across different plugins (eg, for k8s audit logs, we may have different plugins but only one type of `event source`).
+
+The `name` should match this regular expression `^[a-z]+[a-z0-9_]*$`.
+
+## Fields
+
+The `fields` are used for conditions in rules, please describe the available fields of your plugin in the README.
+
+For example:
+
+| Name                     | Type   | Description                    |
+| ------------------------ | ------ | ------------------------------ |
+| `docker.status`          | string | Status of the event            |
+| `docker.id`              | string | ID of the event                |
+| `docker.from`            | string | From of the event (deprecated) |
+| `docker.type`            | string | Type of the event              |
+| `docker.action`          | string | Action of the event            |
+| `docker.stack.namespace` | string | Stack Namespace                |
+
+# Propose your Plugin
+
+It's easy, follow these steps:
+* fork https://github.com/falcosecurity/plugins
+* update `https://github.com/falcosecurity/plugins/blob/master/registry.yaml` in your fork
+* add your plugin in the `plugins.source` list
+* submit the Pull Request with your modification
```

Comment:
Why not propose to use this link :point_down:  instead?

https://github.com/falcosecurity/plugins/edit/master/registry.yaml

At least, it would be useful to a remind about signing commit (you know, we require the DCO).